data/shared/citizen/scripting/v8/main.js

```diff
@@ -100,31 +101,55 @@ const EXT_LOCALFUNCREF = 11;
 	global.addRawEventHandler = global.addRawEventListener;
 
 	// Client events
-	global.addEventListener = (name, callback, netSafe = false) => {
+	const addEventListener = (name, callback, netSafe = false) => {
 		if (netSafe) {
 			netSafeEventNames.add(name);
 		}
 
 		emitter.on(name, callback);
 	};
-	global.on = global.addEventListener;
+
+	global.addEventListener = addEventListener;
+	global.on = addEventListener;
+
+	// global.RegisterNetEvent = function (name) { netSafeEventNames.add(name); };
+	// global.RegisterServerEvent = global.RegisterNetEvent;
+	global.AddEventHandler = addEventListener;
 
 	// Net events
-	global.addNetEventListener = (name, callback) => global.addEventListener(name, callback, true);
+	global.addNetEventListener = (name, callback) => addEventListener(name, callback, true);
 	global.onNet = global.addNetEventListener;
 
 	global.removeEventListener = emitter.off.bind(emitter);
 
+	// Emit
 	global.emit = (name, ...args) => {
 		const dataSerialized = pack(args);
 
 		Citizen.invokeNative('0x91310870', name, dataSerialized, dataSerialized.length);
 	};
-	global.emitNet = (name, ...args) => {
-		const dataSerialized = pack(args);
 
-		Citizen.invokeNative('0x7fdd1128', name, dataSerialized, dataSerialized.length);
-	};
+	global.TriggerEvent = emit;
+
+	if (_in("0xcf24c52e", _r))  { // server
```

Comment:
It's ill advised to use these functions here, as they're part of automatic natives codegen (as in, can change w/o any notice).